klt-cert-manager/Makefile

```diff
@@ -11,9 +10,13 @@ TAG?=latest
 RELEASE_IMAGE:=keptn-cert-operator:$(TAG)
 
 ARCHS = amd64 arm64
+COMMONENVVAR=GOOS=$(shell uname -s | tr A-Z a-z)
+BUILDENVVAR=CGO_ENABLED=0
+
+VERSION=$(shell echo $(RELEASE_VERSION) | awk -F - '{print $$2}')
 
 # ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
-ENVTEST_K8S_VERSION = 1.25.0
+ENVTEST_K8S_VERSION = 1.24.2
```

Comment:
why not using version 25 ?

Reply:
this is to test with lowest compatible cluster